substrate/utils/frame/benchmarking-cli/src/pallet/template.hbs

```diff
@@ -22,7 +22,11 @@ use core::marker::PhantomData;
 
 /// Weight functions for `{{pallet}}`.
 pub struct WeightInfo<T>(PhantomData<T>);
+{{#if (eq pallet "frame_system_extensions")}}
```

Comment:
Looks a bit hacky, see my review comment

Reply:
It doesn't feel hacky to me, it's basically the same as the model for the `frame_system` weights in the other templates. They also get special treatment because of the name, like in `substrate/.maintain/frame-weight-template.hbs`.

Reply:
Yea but its not neccecary here since you can move the TE benchmarks to a `frame-system-ext-benchmarks` crate.